ldi-orchestrator/ldio-common/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/types/LdioVaultTransformer.java

```diff
@@ -0,0 +1,26 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldio.types;
+
+import org.apache.jena.rdf.model.Model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This LdioTransformer can be used for test purposes.
+ */
+public class LdioVaultTransformer extends LdioTransformer {
```

Comment:
Can't this be in a test package?

Reply:
problem is.. Then it can not be shared over other packages, can it?